### DISCUSSION TITLE
Leech 4gb not working

### DISCUSSION THREAD

**Discussion by paulper2** [anasty17/mirror-leech-telegram-bot#1334] [General]
I have recently buyed telegram premium and generated a user session string and restarted the bot still leech is only a 2gb file can you please help me out

**Comment by anasty17** [comment]
Create new session string

**Reply by paulper2** [reply]
did still not working

**Reply by anasty17** [reply]
Join the group on telegram and report there.

**Reply by paulper2** [reply]
Can i have the invite link for the group

**Reply by anasty17** [reply]
https://t.me/mltb_official

It's in channel description

**Reply by paulper2** [reply]
Thanks
